### PR TITLE
[Doc] Add annotations for shared-data

### PR DESCRIPTION
**Pull request by EsoragotoSpirit**
Signed-off-by: EsoragotoSpirit <wanglichen@starrocks.com>

Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

File: docs/administration/Configuration.md [hunk 1]
```diff
@@ -273,6 +273,23 @@ This section provides an overview of the static parameters that you can configur
 | tablet_sched_storage_cooldown_second | -1              | The latency of automatic cooling starting from the time of table creation. The alias of this parameter is `storage_cooldown_second`. Unit: second. The default value `-1` specifies that automatic cooling is disabled. If you want to enable automatic cooling, set this parameter to a value greater than `-1`. |
 | tablet_stat_update_interval_second   | 300             | The time interval at which the FE retrieves tablet statistics from each BE. Unit: second. |
 
+#### StarRocks Shared-date cluster
```
Comment (kevincai) [inline docs/administration/Configuration.md:276]:
```suggestion
#### StarRocks shared-data cluster
```

File: docs/deployment/deploy_shared_data.md [hunk 2]
```diff
@@ -210,7 +249,7 @@ In addition to the regular table PROPERTIES, you need to specify the following P
 
 | **Property**            | **Description**                                              |
 | ----------------------- | ------------------------------------------------------------ |
-| enable_storage_cache    | Whether to enable the local disk cache. Default: `true`.<ul><li>When this property is set to `true`, the data to be loaded is simultaneously written into the object storage and the local disk (as the cache for query acceleration).</li><li>When this property is set to `false`, the data is loaded only into the object storage.</li></ul>**NOTE**<br />To enable the local disk cache, you must specify the directory of the disk in the BE configuration item `starlet_cache_dir`. |
+| enable_storage_cache    | Whether to enable the local disk cache. Default: `true`.<ul><li>When this property is set to `true`, the data to be loaded is simultaneously written into the object storage and the local disk (as the cache for query acceleration).</li><li>When this property is set to `false`, the data is loaded only into the object storage.</li></ul>**NOTE**<br />To enable the local disk cache, you must specify the directory of the disk in the BE configuration item `storage_root_path`. |
```
Comment (kevincai) [inline docs/deployment/deploy_shared_data.md:252]: this is renamed to `datacache.enable` in `main` and `branch-3.1` in PR #25768, please check with PM on this and make corresponding doc update.

Comment (EsoragotoSpirit) [reply docs/deployment/deploy_shared_data.md:252]: Yes, this will be modified in future PRs for 3.1. This PR only covers current branches.

**Review by hellolilyliuyi — APPROVED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.1

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Pull request merged**